english/hospital_bed/hospital_bed.md

```diff
@@ -0,0 +1,8 @@
+Just then, a crazed ostrich smacks into you at full-speed.  Your phone flies out of your hands and you hit the pavement.  
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.